packages/components/Button/src/Button.settings.win32.ts

```diff
@@ -81,6 +81,12 @@ export const settings: IComposeSettings<IButtonType> = [
           borderColor: 'strokeFocus2',
           borderWidth: 2,
         },
+        stack: {
+          // TODO - #984: temp solution to remove extra space for the focused button
```

Comment:
nit: elsewhere we use "GH #984: blah blah" instead of "TODO - #984". 
I only adopted that style myself because my old manager disliked seeing the word "TODO" in code 😅 but I guess we can stay consistent so it's easier to find these?

Reply:
Also instead of the number of this PR, could we file a separate issue on the GitHub issue tracker?

Reply:
I reported a new bug: https://github.com/microsoft/fluentui-react-native/issues/986
And added GH instead of TODO